src/domain/moit/TimeSelectBottomSheet.tsx

```diff
@@ -12,15 +13,42 @@ import { useSelectTime } from './hooks/useSelectTime';
 
 interface Props {
   modalProps: ModalProps;
+  testH: number;
+  testM: number;
+  testHSetter: (n: number) => void;
+  testMSetter: (n: number) => void;
 }
 
-// TODO: 타입 변환시 해당 인덱스로 스크롤 로직 추가
-// TODO: 인풋 로직 개발되면 저장버튼에 싱크 로직 추가
-
-const TimeSelectBottomSheet = ({ modalProps }: PropsWithChildren<Props>) => {
+const TimeSelectBottomSheet = ({
+  modalProps,
+  testHSetter,
+  testH,
+  testM,
+  testMSetter,
+}: PropsWithChildren<Props>) => {
   const [currentCursor, setCurrentCursor] = useState<TimeZoneCursor>('start');
   const { hour, min, startTime, endTime } = useSelectTime(currentCursor);
 
+  // 마운트 시점 스크롤. 지연 로직을 추가하지 않으면, 정상 동작하지 않음
+  useEffectOnce(() => {
+    setTimeout(() => {
+      hour.ref.current?.scrollTo(0, Math.floor((testH + 1) * 52));
+      min.ref.current?.scrollTo(0, Math.floor((testM + 1) * 52));
+    }, 300);
+  });
+
+  // 커서 변경시 스크롤 로직.
+  // 더 좋은 로직이 있는건 확실한디... :(
+  useEffect(() => {
+    setTimeout(() => {
+      hour.ref.current?.scrollTo(0, hour.selectedIndex * 52);
+      min.ref.current?.scrollTo(0, min.selectedIndex * 52);
+    });
+
+    //! don't delete this ignore.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currentCursor]);
```

Comment:
```suggestion
  useEffect(() => {
    setTimeout(() => {
      hour.ref.current?.scrollTo(0, hour.selectedIndex * 52);
      min.ref.current?.scrollTo(0, min.selectedIndex * 52);
    });

    //! don't delete this ignore.
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [currentCursor]);
```

여기 setTimeout이 왜 필요하징??

Reply:
콜스택 마지막에 넣어서 비동기로 실행하려구 !!